README.adoc

```diff
@@ -414,68 +416,64 @@ endif::[]
 
 // Cloud-hosted guide instruction
 ifdef::cloud-hosted[]
-Run the following command to get the node IP for the `system` service:
+In this execise, you need to access the services by using Kubernetes API.
+Run the following command to start a proxy to the Kubernetes API server:
 
 ```
-kubectl describe pod system | grep Node
+kubectl proxy
 ```
 {: codeblock}
 
-The output shows the node IP that is later used to access the service. 
-It appears in a format similar to the following:
-
-```
-Node:         10.114.85.140/10.114.85.140
-Node-Selectors:  <none>
+Open another command-line session by selecting **Terminal** > **New Terminal** from the menu of the IDE.
+Run the following commands to store the proxy path of the **system** and **inventory** services.
 ```
-
-The IP for the system service is `10.114.85.140`.
-Store the IP in a variable.
-
-```
-SYSTEM_HOST={system-node-ip}
+NAMESPACE_NAME=`bx cr namespace-list | grep sn-labs- | sed 's/ //g'`
+SYSTEM_PROXY=localhost:8001/api/v1/namespaces/$NAMESPACE_NAME/services/system-service/proxy
+INVENTORY_PROXY=localhost:8001/api/v1/namespaces/$NAMESPACE_NAME/services/inventory-service/proxy
 ```
 {: codeblock}
 
-Use another `kubectl` command to get the node IP for the `inventory` service:
+Run the following echo commands to verify the variables:
 
 ```
-kubectl describe pod inventory | grep Node
+echo $SYSTEM_PROXY && echo $INVENTORY_PROXY
 ```
 {: codeblock}
 
-Store this IP in a variable as well.
+
+The output appears similar to the following:
 
 ```
-INVENTORY_HOST={inventory-node-ip}
+localhost:8001/api/v1/namespaces/sn-labs-yourname/services/system-service/proxy
+localhost:8001/api/v1/namespaces/sn-labs-yourname/services/inventory-service/proxy
 ```
-{: codeblock}
 
-Verify that the variables that contain the IPs are set correctly:
+Then use the following **curl** command to access your **system** microservice:
 
 ```
-echo $SYSTEM_HOST && echo $INVENTORY_HOST
+curl http://$SYSTEM_PROXY/system/properties | jq
 ```
 {: codeblock}
 
-Then use the following `curl` commands to access your microservices:
+and use the following **curl** command to access your **inventory** microservice:
 
 ```
-curl http://$SYSTEM_HOST:31000/system/properties
+curl http://$INVENTORY_PROXY/inventory/systems/system-service | jq
 ```
 {: codeblock}
 
+The first URL returns system properties and the name of the pod in an HTTP header called **X-Pod-Name**.
```

Comment:
Change "The first URL" to "The `http://$SYSTEM_PROXY/system/properties` URL" to avoid confusion regarding what URL is being referenced. Also, change "HTTP header called" to "HTTP header that is called".